### ISSUE TITLE
Accessibility Issue : Search Filter Results Web Part not tabbing between filters for radio (checkbox template) control element.

### ISSUE THREAD

**Issue by CollinHunterGOC** [microsoft-search/pnp-modern-search#3259]
**Version used**
Ex: 4.9.3

**Describe the bug**
When the PnP - Search Filters Web Part is configured/customized with the Check Box template and Multi-values deselected, the web part will display radio buttons (instead of checkboxes) and the area of the PnP - Search Filters Web Part is ultimately ignored when the user tries to cycle within the filter options using the Tab key of the keyboard.

The problem is due to fact that the Fluent UI library doesn't support properties to handle tabbing between the Choice Group control option elements. Only Checkbox controls work. I checked the current version (v9.0) and the version of this web part (v8.109.7) and the Choice Group field doesn't support the TabIndex property and therefore doesn't work. You can even go to the development website and try the sample there.

In Version 3, if you selected the Check Box template, it displayed Check Boxes and it worked. I suspect Version 3 also didn't use the FluentUI library and it is architected differently.

**For the developers:** is it possible to replace the Choice Group control with the Check box control regardless whether multi-select is selected or not?

Code:  /pnp-modern-search/search-parts/src/components/filters/FilterCheckBoxComponent.tsx:

```ts
export class FilterCheckBoxComponent extends React.Component<IFilterCheckBoxProps, IFilterCheckBoxState> {...

 _if (this.props.isMulti) {
            renderInput = <Checkbox
                styles={{
                    root: {
                        padding: 10,
                    },
                    label: {
                        width: '100%'
                    },
                    text: {
                        color: this.props.count && this.props.count === 0 ? this.props.themeVariant.semanticColors.disabledText : textColor
                    }
                }}
                theme={this.props.themeVariant as ITheme}
                defaultChecked={this.props.selected}
                disabled={this.props.disabled}
                title={filterValue.name}
                label={filterValue.name}
                onChange={(ev, checked: boolean) => {
                    filterValue.selected = checked;
                    this.props.onChecked(this.props.filterName, filterValue);
                }}
                onRenderLabel={(props, defaultRender) => {
                    return <Text block nowrap title={props.label}>{props.label}</Text>;
                }}
            />;
        } else {
            renderInput = <ChoiceGroup
                styles={{
                    root: {
                        position: 'relative',
                        display: 'flex',
                        paddingRight: 10,
                        paddingLeft: 10,
                        paddingBottom: 7,
                        paddingTop: 7,
                        selectors: {
                            '.ms-ChoiceField': {
                                marginTop: 0
                            }
                        }
                    }
                }}_
```

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Go to the Search Filters Results Web Part.
2. Modify the Search Filters Results Web Part and configure the filter using a Check Box template and make sure Multi-select is not selected.'
3. Republish the page.
4. Try to tab between elements in the PnP - Search Filters area.
5. The user will not be able to tab to the PnP - Search Filters area and therefore by pass it.

**Expected behavior**
When a user tries to filter search results and the search results filter web part has radio button options, the user should be able to tab through all available options using a keyboard.

**Screenshots**

**Desktop (please complete the following information):**
 - Browser [Chrome and Edge]

**Additional context**

**Comment by wobba** [comment]
Can you show a screenshot of what the change would look like? And maybe do a pull request for the change?

**Comment by CollinHunterGOC** [comment]
export class FilterCheckBoxComponent extends React.Component<IFilterCheckBoxProps, IFilterCheckBoxState> {

    public render() {

        let filterValue: IDataFilterValueInfo = {
            name: this.props.name,
            value: this.props.value,
            selected: this.props.selected
        };

        let renderInput: JSX.Element = null;
        let textColor: string = this.props.themeVariant && this.props.themeVariant.isInverted ? (this.props.themeVariant ? this.props.themeVariant.semanticColors.bodyText : '#323130') : this.props.themeVariant.semanticColors.inputText;

            renderInput = <Checkbox
                styles={{
                    root: {
                        padding: 10,
                    },
                    label: {
                        width: '100%'
                    },
                    text: {
                        color: this.props.count && this.props.count === 0 ? this.props.themeVariant.semanticColors.disabledText : textColor
                    }
                }}
                theme={this.props.themeVariant as ITheme}
                defaultChecked={this.props.selected}
                disabled={this.props.disabled}
                title={filterValue.name}
                label={filterValue.name}
                onChange={(ev, checked: boolean) => {
                    filterValue.selected = checked;
                    this.props.onChecked(this.props.filterName, filterValue);
                }}
                onRenderLabel={(props, defaultRender) => {
                    return <Text block nowrap title={props.label}>{props.label}</Text>;
                }}
            />;
        }

        return renderInput;
    }
}

**Comment by wobba** [comment]
Sure you don't want to try a PR? 🙂 See https://microsoft-search.github.io/pnp-modern-search/how-to-contribute/

**Comment by CollinHunterGOC** [comment]
Please scrap my idea. Apparently, one of my colleagues just tried the Check box template with multi-select in both v.4.7.0 and v4.9.3 and tabbing still doesn't work. Should I file a separate bug? The only option that seems to work is the combo box,

**Comment by CollinHunterGOC** [comment]
I will keep digging. Perhaps the multi-values is causing an issue with the checkboxes.

**Comment by wobba** [comment]
Just leave this issue.
